capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] Here we have to only Capitalize the first letter of the given name. so kindly use 
`@nameable.correct_name.capitalize`